app/log/loki/client.go

```diff
@@ -101,10 +104,15 @@ func (c *Client) Run() {
 	for {
 		select {
 		case line := <-c.input:
+			if len(line) > c.maxLogLineLen && c.maxLogLineLen != 0 {
+				continue // Silently drop log line longer than maxLogLineLen if set
```

Comment:
not sure about silent, what about using a log filter?

Reply:
I'm doing what `batch` does pretty much, we discussed this briefly on Slack and you suggested to just drop it.

What behavior would you like to see instead?

Reply:
well, a log it using a log filter

but sure, this is only loki client itself, which is not our primary logging framework, it is best effort, so yeah, maybe silent drop is fine.